test/features/resource_availability.feature

```diff
@@ -1,252 +1,113 @@
-        Then I execute the script "document.getElementById('field-image-url').value='http://ckanext-data-qld.docker.amazee.io/'"
-        Then I fill in "name" with "Resource created by <User> with default values"
-        Then I fill in "description" with "description"
-        Then I press "save"
+        And I go to "/dataset/contains-de-identified-data-no-visibility-false-acknowledgment-no"
+        And I wait for 1 seconds
+        And I press the element with xpath "//a[@title='Hide Resource']"
+        Then I should see an element with xpath "//th[contains(text(), 'Resource visible')]"
+        And I should see an element with xpath "//th[contains(text(), 'Re-identification risk governance completed?')]"
 
-          Examples: Users
+        Examples: Users
             | User          |
-            | SysAdmin      |
             | TestOrgAdmin  |
             | TestOrgEditor |
 
-    ###
-    # Check the resource  with default values
-    ###
-    Scenario Outline: Check the resource with default values
+    Scenario: Unauthenticated user should not see a hidden visible resource when de-identified data is NO and Resource visibility is FALSE and Acknowledgement is NO
+        Given "Unauthenticated" as the persona
+        When I go to "/dataset/contains-de-identified-data-no-visibility-false-acknowledgment-no"
+        And I wait for 1 seconds
+        Then I should not see an element with xpath "//a[@title='Hide Resource']"
+
+
+    Scenario Outline: Organisation users should see a visible resource when de-identified data is YES and Resource visibility is TRUE and Acknowledgement is YES
         Given "<User>" as the persona
         When I log in
-        And I go to "/contains-de-identified-data-yes-visibility-true-acknowledgment-no"
-        And I wait for 3 seconds
-        Then I press the element with xpath "//a[@title='Resource created by <User> with default values']"
-        Then I should see "Resource visible"
-        Then I should see "Re-identification risk governance completed?" 
-        
-
-          Examples: Users
+        And I go to "/dataset/contains-de-identified-data-yes-visibility-true-acknowledgment-yes"
+        And I wait for 1 seconds
+        And I press the element with xpath "//a[@title='Show Resource']"
+        Then I should see an element with xpath "//th[contains(text(), 'Resource visible')]"
+        And I should see an element with xpath "//th[contains(text(), 'Re-identification risk governance completed?')]"
+
+        Examples: Users
             | User          |
-            | SysAdmin      |
             | TestOrgAdmin  |
             | TestOrgEditor |
+
+    Scenario: Unauthenticated user should see a visible resource when de-identified data is YES and Resource visibility is TRUE and Acknowledgement is YES
+        Given "Unauthenticated" as the persona
+        When I go to "/dataset/contains-de-identified-data-yes-visibility-true-acknowledgment-yes"
+        And I wait for 1 seconds
```

Comment:
Is this wait essential? Normally the system is pretty good at waiting until the page has loaded without explicit instructions.

Reply:
When running the tests locally in the past they have been required, especially when JS needed to execute after the form was loaded.
I removed the steps and the tests all passed locally.